pyproject.toml

```diff
@@ -9,7 +9,7 @@ packages = [
     {include = "**/*.py", from = "src"},
 ]
 readme = "README.md"
-version = "0.10.1"
+version = "0.10.2"
```

Comment:
I'd say we should call this `0.11` - not because it's backwards incompatible, but because if somebody asks "what version of the SDK do I need to use UR Cap?" it's a bit nicer to say "0.11" than "you need at least "0.10.2".